### DISCUSSION TITLE
Allow self-registration: when I registrated with an account that conform to the rules, it warns the mail confiquration is incomplete or invalic

### DISCUSSION THREAD

**Discussion by HeeOne** [requarks/wiki#6155] [Help / Questions] [answered]
![image](https://user-images.githubusercontent.com/15077277/218355782-2872ca02-b713-4341-9b1e-55a7c4e0cfac.png)

**Comment by NGPixel** [comment, marked as answer]
You need to configure your mail server settings under Admin > Mail.
